### PR TITLE
added hashRandomTransformer, and bounds checks to int and floats

### PR DESCRIPTION
**Pull request by crxpandion**

### CODE REVIEW

File: transformer/redshift_types.go [hunk 1]
```diff
@@ -68,9 +70,9 @@ func GetTransform(t_type string) ColumnTranformer {
 // New types should register here
 var (
 	transformMap = map[string]ColumnTranformer{
-		"int":          intFormat,
-		"bigint":       intFormat,
-		"int8":         intFormat,
+		"int":          intFormat(32),
+		"bigint":       intFormat(64),
+		"int8":         intFormat(8),
```
Comment (aasted) [inline transformer/redshift_types.go]: int8 is big int. This should read intFormat(64)

Comment (aasted) [inline transformer/redshift_types.go]: http://docs.aws.amazon.com/redshift/latest/dg/c_Supported_data_types.html

**Comment by drewww** [comment]
Awesome, thanks for knocking this out! LGTM but I can barely read Go. :speak_no_evil: 

That minor note about the additive nature of the hash function seems important to me and should probably be made clear somewhere other than a comment on the test. So the hash is deterministic but not in its input, but rather on that input + the entire state of the md5 object over the lifetime of the process? This is totally fine for our use case now, but could be a gotcha if someone just looks at the name and thinks they're getting a deterministic hasher (which could plausibly be useful too, though not something we actually need here).

File: transformer/redshift_types.go [hunk 2]
```diff
@@ -22,7 +24,7 @@ import (
 // for time transformers. Transform generators allow the user to define how
 // the transformer should parse a inbound property.
 
-// Note: all times are in PST.
+// All times are in PST.
```
Comment (ossareh) [inline transformer/redshift_types.go]: this isn't golint compatible, I think you guys should start making these comments golint compatible - godoc is so good.

File: transformer/redshift_types.go [hunk 3]
```diff
@@ -68,9 +70,8 @@ func GetTransform(t_type string) ColumnTranformer {
 // New types should register here
 var (
 	transformMap = map[string]ColumnTranformer{
-		"int":          intFormat,
-		"bigint":       intFormat,
-		"int8":         intFormat,
+		"int":          intFormat(32),
+		"bigint":       intFormat(64),
```
Comment (ossareh) [inline transformer/redshift_types.go]: Is there a practical benefit to having both forms of int? Why not just make everything 64bit?

File: transformer/redshift_types.go (hunk 2 above)
Comment (crxpandion) [inline transformer/redshift_types.go]: agree - will be in a separate PR

**Comment by crxpandion** [comment]
@drewww The most recent commit makes the hashes consistent. The hashes were consistent previously just it wasnt obvious to me that they were.
